pkg/roachprod/fluentbit/fluentbit.go

```diff
@@ -71,6 +70,7 @@ func Install(ctx context.Context, l *logger.Logger, c *install.SyncedCluster, co
 			DatadogSite:    config.DatadogSite,
 			DatadogAPIKey:  config.DatadogAPIKey,
 			DatadogService: config.DatadogService,
+			Hostname:       vm.Name(c.Name, int(node)),
```

Comment:
What will these hostnames look like? Hopefully globally unique?

Reply:
The format is `CLUSTER-HOST`. For example `drt-large-0001`, `drt-large-0002`, etc. Technically not globally unique, but `roachprod` doesn't allow you to create multiple clusters of the same name.